test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -118,6 +118,19 @@ public static Person generateTestPerson() {
         }
     }
 
+    /*
+    Overloaded method to generate a person
+     */
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Generates a person …` instead of `Overloaded method to...` or `This method is for …`). You don't have to explain that it's overloaded, the programmer should be able to tell :)